README.md

```
@@ -24,18 +24,24 @@
     - [x] 직선 한 칸 이동할 수 있다.
     - [x] 직선 한 칸 앞에 같은 편 기물이 있을 경우, 움직일 수 없다.
     - [x] 직선 한 칸 앞에 상대 편 기물이 있을 경우, 그 기물을 제거 후 움직일 수 있다.
+    - [x] 궁성 내부 중, 대각 이동이 가능한 위치라면 대각선으로 이동할 수 있어야 한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)